samples/tfm/tfm_hello_world/prj.conf

```diff
@@ -4,5 +4,5 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-CONFIG_TFM_PROFILE_TYPE_MINIMAL=y
+CONFIG_TFM_PROFILE_TYPE_NOT_SET=y
```

Comment:
Was that intentional? Do we need to stop using the minimal profile for this?

Reply:
Sample uses tries to demonstrate hash functionality with CONFIG_TFM_CRYPTO_HASH_MODULE_ENABLED, which cannot be activated with CONFIG_TFM_PROFILE_TYPE_MINIMAL.

Reply:
tfm_hello_world does hashing, which needs the crypto partition, and which is not included in minimal profile. I don't know how this could have worked before

Reply:
The sample uses hashing conditionality based on the  TFM_CRYPTO_HASH_MODULE_ENABLED. The sample.yaml file enables that only for some configurations. I would highly suggest to change that back to profile minimal in the default configuration. The TF-M minimal is production ready and the profile not-set it is not so this is a major change that should not be relevant to adding support for 9151 I think.

Reply:
Restored the minimal and removed the hash from sample output.